packages/inngest/src/utils/sanity-client.ts

```diff
@@ -0,0 +1,8 @@
+import {createClient} from '@sanity/client'
+
+export const sanityClient = createClient({
```

Comment:
I wanted to import and use sanityClient from `skill-lesson` pkg but that'd be a cyclic dependency :|

Reply:
we have this exact file exporting `sanityClient` in every app and many/most packages, could we create a super-small `sanity-client` package that does this once and then each app can import it. I think each app's individual `process.env` would be applied.

Reply:
the inngest package should be nuked